src/tracing/types.rs

```diff
@@ -353,31 +373,44 @@ impl CallTraceNode {
                 to: self.trace.address,
                 value: self.trace.value,
                 gas: self.trace.gas_limit,
-                input: self.trace.data.clone(),
+                input: Bytes::new(),
+                // input: self.trace.data.clone(),
                 call_type: self.kind().into(),
                 // tx_type: self.trace.tx_type,
             }),
             CallKind::Create | CallKind::Create2 => Action::Create(CreateAction {
                 from: self.trace.caller,
                 value: self.trace.value,
                 gas: self.trace.gas_limit,
-                init: self.trace.data.clone(),
+                init: Bytes::new(),
+                // init: self.trace.data.clone(),
```

Comment:
for a create call (contract deployment), it's fine to show this in the trace, so can go back to initial